fe/fe-core/src/test/java/com/starrocks/http/ShowMetaInfoActionTest.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.http;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.starrocks.catalog.Database;
+import com.starrocks.catalog.OlapTable;
+import com.starrocks.common.Config;
+import com.starrocks.server.GlobalStateMgr;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.junit.Assert.assertTrue;
+
+public class ShowMetaInfoActionTest extends StarRocksHttpTestCase {
+
+    private static final long DB_ID = 1000 + testDbId;
+    private static final String DB_NAME = "TEST_DB";
+    private static final String TABLE_NAME = "TEST_TABLE";
+    private static final long EXPECTED_SINGLE_REPLICA_SIZE = 1024000L;
```

Comment:
this expected size is a little hard to understand why it is this specific number, either get it from internal data structure, or some comments to let others know why it is this number.